kb-survey/src/environments/environment.ts

```diff
@@ -0,0 +1,16 @@
+// This file can be replaced during build by using the `fileReplacements` array.
+// `ng build` replaces `environment.ts` with `environment.prod.ts`.
+// The list of file replacements can be found in `angular.json`.
+
+export const environment = {
```

Comment:
Don't push any of the environment files to Git Hub.  create an environment.sample.ts and put the base structure of environment inside that file

Reply:
okay